src/components/20-molecules/input-phone/index.react.d.ts

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export interface AXAInputPhoneProps {
```

Comment:
Pls. make the `label` and `errorprefix` optional, and add `on*()` methods, so that it can be integrated into React.
Thanks!
See: https://github.com/axa-ch-webhub-cloud/pattern-library/blob/develop/src/components/10-atoms/input-text/index.react.d.ts

Reply:
(also the `value` property of couse, thanks)